docs/using-injected-bindings/template.md

```diff
@@ -3,3 +3,52 @@ sidebar_position: 1
 ---
 
 # Template
+
```

Comment:
Can you explain how this story goes under "Template"? What is the meaning of template here?